data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -242,6 +245,14 @@ podman run -d --name ironic-ramdisk-logs \
      --entrypoint /bin/runlogwatch.sh \
      -v $IRONIC_SHARED_VOLUME:/shared:z ${IRONIC_IMAGE}
 
+# Failure to start image-customization results in a very confusing error
+icc_id=$(podman ps --filter name=image-customization --filter status=running --format '{{`{{.ID}}`}}')
```

Comment:
Shouldn't be `... --format '{{.ID}}')` ?

Reply:
Nope. This file is already a Go template, we need to escape {{ in it. That's the way I found on the internet.

Reply:
Oh yes that's right, I've missed that, thanks